.nancy-ignore

```diff
@@ -1,4 +1,5 @@
 CVE-2018-1714 # old helm issue that has all the tickets closed so not much we can do about it
 CVE-2022-23328 # Geth is able to be used for DoS attacks on different nodes. Not something we particularly care about for our tests.
 sonatype-2021-0076 # More weird geth vulnerabilities we don't care about.
-CVE-2022-31030  # This is associated to the dependency 'containerd' and also warns of a DoS attack.g
\ No newline at end of file
+CVE-2022-31030  # This is associated to the dependency 'containerd' and also warns of a DoS attack.
+CVE-2022-37450 # Geth bug that only affects nodes used for mining
```

Comment:
would it affect the reorg tests anyhow?

Reply:
We would need to explicitly try and exploit it. And even if we did 🤷